API.md

```diff
@@ -0,0 +1,204 @@
+
+#### Query parameters
+
+* `topojson` - if present, the location's topojson structure will be sent with the object.
+
+### `GET /locations/regions`
+
+Lists all locations that are regions
+
+#### Query parameters
+
+* `topojson` - if present, the location's topojson structure will be sent with the object.
+
+## World Bank Demographic data (WB Extra)
+
+### `GET /wb_extra`
+
+Retrives World Bank demographic information (population and gdp) per country, per year.
+
+#### Query parameters
+
+* `startYear`
+* `endYear`
+
+#### Observations
+
+Collection is serialized as an object with country codes as keys.
+
+### `GET /wb_extra/:iso`
+
+Retrives World Bank demographic information (population and gdp) per country, per year.
+
+#### Query parameters
+
+* `startYear` - show results from this year onwards
+* `endYear` - show results up to this year
+
+## Historical Emissions
+
+### `GET /emissions/meta`
+
+Retrieves metadata of the historical emissions dataset. This metadata is the list of `data_sources`, `sectors`, `gases`, `locations` and `gwps`. All these parameters are filters in the `/emissions` endpoint
+
+#### Observations
+
+Collection is serialized as an object with the metadata entity as key.
+
+### `GET /emissions`
+
+Retrieves time series data for historical emissions.
+
+#### Query parameters
+
+* `source` - emission data source id (cait, pik, wb)
+* `gas` - emission gas id
+* `sector` - emission sector id
+* `gwp` - emission gwp id
+* `location` - location iso3 code
+* `date_after` - show results from this year onwards
+* `date_before` - show results up to this year
+
+Note: all these parameters accept multiple values, separated by commas.
+
+#### Observations
+
+`source` query parameter is mandatory, together with at least one of `location`, `sector` or `gas`.
+
+## NDCs (texts)
+
+### `GET /ndcs/text`
+
+Lists all known NDC texts
+
+#### Query parameters
+
+* `query` - Free text search (lists snippets)
+
+The following parameters can be used to search sdg linkages' snippets of text (lists snippets in the original text).
+
+* `target` - NdcSdg target number
+* `goal` - NdcSdg goal number
+* `sector` - NdcSdg sector number
+
+### `GET /ndcs/:iso/text`
+
+Shows NDC texts for the country given by the :iso parameter
+
+#### Query parameters
+
+* `query` - Free text search (highlights snippets in the original text)
+
+The following parameters can be used to search and highlight sdg linkages' snippets of text
+
+* `target` - NdcSdg target number
+* `goal` - NdcSdg goal number
+* `sector` - NdcSdg sector number
+
+## NDCs (SDG linkages)
+
+### `GET /ndcs/sdgs`
+
+Shows the relations between all SDG metadata (targets, goals and sectors)
+
+#### Observations
+
+Collection is serialized as an object with the metadata entity as key
+
+### `GET /ndcs/:iso/sdgs`
+
+Shows linkage data for the given country code.
+
+### `GET /ndcs/sdgs_overview`
+
+Shows an overview of linkage data to meet specific front-end requirements:e .
```

Comment:
there's a trailing `:e .` I guess it's a vim artefact =)

Reply:
Derp.